style.css

```diff
@@ -0,0 +1,657 @@
+body {
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+nav {
+  display: flex;
+  color: #4f545d;
+  justify-content: space-between;
+  margin: 0px 6% 0px 6%;
+  align-items: center;
+  margin-top: 10px;
+}
+
+nav ul {
+  list-style-type: none;
+  display: flex;
+  margin-right: 50px;
+  padding-right: 80px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-family: "Nunito", sans-serif;
+  font-weight: bolder;
+  font-size: small;
+}
+
+nav ul li {
+  margin: 0 20px 0 20px;
+}
+
+nav .logo {
+  order: -1;
+  margin-left: 10%;
+}
+
+nav section img {
+  width: 80%;
+}
+
+nav ul .signup-btn {
+  padding: 5px 10px;
+  border: 2px solid black;
+  border-radius: 5px;
+  cursor: pointer;
+  display: inline-block;
+  transition: background-color 0.3s ease;
+  background-color: white;
+}
+nav ul .signup-btn:hover {
+  background-color: goldenrod;
+  border-color: goldenrod;
+}
+
+nav ul .language-dropdown {
+  position: relative;
+  padding: 8px 15px;
+  cursor: pointer;
+}
+
+nav ul .language-dropdown .dropdown-content {
+  display: none;
+  position: absolute;
+  background-color: #fff;
+  box-shadow: 0 8px 16px rgba(0, 0, 0, 0.1);
+  z-index: 1;
+}
+
+nav ul .language-dropdown:hover .dropdown-content {
+  display: block;
+}
+
+nav ul .language-dropdown .dropdown-content li {
+  padding: 10px;
+  text-align: center;
+  border-bottom: 1px solid #ddd;
+}
+
+nav ul .language-dropdown .dropdown-content li:last-child {
+  border-bottom: none;
+}
+main {
+  display: flex;
+  margin-bottom: 200px;
+}
+
+.left-section {
+  padding: 0 0 0 8%;
+  margin: 0% 0% 0% 7%;
+  position: relative;
+  max-width: 35%;
+}
+
+.right-section img {
+  position: absolute;
+  top: 5em;
+  left: 39em;
+  z-index: 1;
+  width: 40%;
+  margin-right: 50%;
+}
+main span h5 {
+  margin-top: 17%;
+  margin-bottom: 0%;
+  color: #e27660;
+  font-family: "Poppins", sans-serif;

```

Comment:
for complete website we are using same font family don't repeat, you can apply the font for body or u can take container class and specify there

Reply:
resolved